utils/constants.ts

```diff
@@ -55,8 +55,8 @@ export const URLS = {
     snek: 'https://squid.subsquid.io/snekk/graphql',
     snekRococo: 'https://squid.subsquid.io/sneck/graphql',
     marck: 'https://squid.subsquid.io/marck/graphql',
-    stick: 'https://squid.subsquid.io/stick/graphql',
-    speck: 'https://squid.subsquid.io/speck/graphql',
+    stick: 'https://squid.subsquid.io/stick/v/v5/graphql',
+    speck: 'https://squid.subsquid.io/speck/v/v5/graphql',
```

Comment:
It should not be a fixed version.

Reply:
ok then what to do?

it isn't deployed on stick/graphql

Reply:
> when we release this feature

Can do a release if everything is good

cc @prury

Reply:
> > when we release this feature
> 
> Can do a release if everything is good
> 
> cc @prury

will check this one again in a few moments